dhis-2/dhis-e2e-test/src/test/java/org/hisp/dhis/tracker/importer/ImportStrategyTests.java

```diff
@@ -157,22 +159,15 @@ public void shouldDeleteWithOnlyIdInThePayload()
         String enrollmentId = response.extractImportedEnrollments().get( 0 );
         String teiId = response.extractImportedTeis().get( 0 );
 
-        trackerActions
-            .postAndGetJobReport( new JsonObjectBuilder().addProperty( "enrollment", enrollmentId ).wrapIntoArray( "enrollments" ),
-                new QueryParamsBuilder().add( "importStrategy=DELETE" ) )
-            .validateSuccessfulImport()
-            .validate().body( "stats.deleted", Matchers.equalTo( 1 ) );
-
-        // todo uncomment these validations when DHIS2-12685 is fixed
-        /*
-        trackerActions.postAndGetJobReport( new JsonObjectBuilder().addProperty( "event", eventId ).wrapIntoArray("events"), new QueryParamsBuilder().add( "importStrategy=DELETE" ) )
-            .validateSuccessfulImport()
-            .validate().body( "stats.deleted", Matchers.equalTo( 1 ) );
+        Consumer<JsonObject> deleteAndValidate = ( payload ) -> {
+            trackerActions.postAndGetJobReport( payload, new QueryParamsBuilder().add( "importStrategy=DELETE" ) )
+                .validateSuccessfulImport()
+                .validate().body( "stats.deleted", Matchers.equalTo( 1 ) );
+        };
 
-        trackerActions.postAndGetJobReport( new JsonObjectBuilder().addProperty( "trackedEntity", teiId ).wrapIntoArray("trackedEntities"), new QueryParamsBuilder().add( "importStrategy=DELETE" ) )
-            .validateSuccessfulImport()
-            .validate().body( "stats.deleted", Matchers.equalTo( 1 ) );
-         */
+        deleteAndValidate.accept( new JsonObjectBuilder().addProperty( "event", eventId ).wrapIntoArray( "events" ));
+        deleteAndValidate.accept( new JsonObjectBuilder().addProperty( "enrollment", enrollmentId ).wrapIntoArray( "enrollments" ));
```

Comment:
Something strange with the code style here? Missing a space before the last end paranthesis?